presto-verifier/src/main/java/com/facebook/presto/verifier/PrestoVerifyCommand.java

```diff
@@ -31,6 +32,13 @@
 public class PrestoVerifyCommand
         extends AbstractVerifyCommand
 {
+    @Override
+    public Module getVerifierModule(SqlParserOptions sqlParserOptions,
```

Comment:
Do you need an another extension point? Have you considered using an existing one? (for example `getAdditionalModules`)?

Or maybe simply add an another method `getClientInfo`.

Reply:
as for naming and functionality, i feel it is better to have a separate module.

fields in clientinfo are retrieved from config.properties and VerifierConfig class has not initialized yet during bootstrap. Is there a way we can get those info at bootstrap step?